server.go

```diff
@@ -803,33 +804,75 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 	// replicate it, and instead it'll only be stored locally.
 	chanGraph := dbs.GraphDB.ChannelGraph()
 
+	// Fetch the source node from the channel graph. If a source node is
+	// found, we'll use it's values for our initial node announcement.
```

Comment:
```suggestion
	// found, we'll use its values for our initial node announcement.
```

Reply:
Thank you for bringing that to my attention.